src/snapred/backend/service/ReductionService.py

```diff
@@ -53,23 +50,17 @@ def __init__(self):
         self.registerPath("save", self.saveReduction)
         self.registerPath("load", self.loadReduction)
         self.registerPath("hasState", self.hasState)
+        self.registerPath("getUniqueTimestamp", self.getUniqueTimestamp)
         return
 
     @staticmethod
     def name():
         return "reduction"
 
-    @FromString
-    def fakeMethod(self):  # pragma: no cover
-        # NOTE this is not a real method
-        # it's here to be used in the registered paths above, for the moment
-        # when possible this and its registered paths should be deleted
-        raise NotImplementedError("You tried to access an invalid path in the reduction service.")
-
     @FromString
     def reduction(self, request: ReductionRequest):
         """
-        Perform reduction on a list of run numbers, once for each grouping in this state.
+        Perform reduction on a single run number, once for each grouping in this state.
```

Comment:
This answers my question from earlier. :+1:

Reply:
I had run this by Michael initially.  He wanted the reduction to go one run at a time.  Note that in the GUI we can specify a list of run numbers, and "continue" on that at once, and then that spawns multiple reduction requests (in a loop).

Reply:
Gotcha.